definitions/ext-ups/dashboard.json

```diff
@@ -184,7 +184,7 @@
 						"nrqlQueries": [
 							{
 								"accountId": 0,
-								"query": "FROM Metric SELECT latest(kentik.snmp.upsAdvBatteryTemperature) AS 'Celsius', (latest(kentik.snmp.upsAdvBatteryTemperature)*1.8) + 32 AS 'Fahrenheit' WHERE provider = 'kentik-ups'"
```

Comment:
This query without the comma isn't valid I think

Reply:
Hi @IreneMLC apologies, these must have been changed when I ran the sanitize dashboards functionality, they are not connected to my project so we can ignore these 😬

Reply:
have reverted the changes and pushed again :)

Reply:
No problem, thanks for the revert!